orc8r/cloud/go/module.mk

```diff
@@ -43,9 +43,9 @@ gen::
 #
 # For example
```

Comment:
Fix the `MODULE.SERVICE.swagger.v1.yml` line as well

Reply:
Oops good catch